solaris/solaris10/generate_wazuh_packages.sh

```diff
@@ -146,7 +147,7 @@ compute_version_revision()
 
 clone(){
     cd ${CURRENT_PATH}
-    git clone $REPOSITORY ${SOURCE}
+    git clone $REPOSITORY ${SOURCE} || clean
```

Comment:
`clean` should be executed outside and it **must** return an exit code.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/76a104533e2ad72b5427a127d56a91bd1f144c6e.